src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -282,7 +296,14 @@ private ExitCode AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersF
             }
             catch (Exception exception)
             {
-                logger.LogError(exception, "An exception occurred while analyzing a template");
+                if (parametersFilePath != null)
+                {
+                    logger.LogError(exception, $"An exception occurred while analyzing template {templateFilePath.Name} with parameters file {parametersFilePath.Name}");
```

Comment:
Oh, sorry, I just realized that the message could be unclear if there are multiple folders, can you use FullName instead of Name in both variables here and in line 305?

Reply:
Changed in [`c62c0d7` (#297)](https://github.com/Azure/template-analyzer/pull/297/commits/c62c0d7cd7fcfd0904a120161940a49bdff76322)